syntax = "proto3";

message TracepointInfo {
  string name = 1;
}